docs/guides/node-operators.md

```diff
@@ -0,0 +1,186 @@
+
+   That's it, voilà! You have officially registered one of the PoLido Operators!
+
+### Pre-staked (existing) Validator
+If you're joining with a pre-staked one, you should interact with `joinOperator` function in NodeOperatorRegistry 
+contract. Import INodeOperator interface to remix and attach it to the `NODE_OPERATOR_REGISTRY_PROXY` address. 
+After the voting process is over, the DAO will create a new operator for you by calling `addOperator`  (Shard Labs owns 
+the private key with DAO role during testnet phase).
+
+Next thing to do is to approve your NFT staking token to the newly created Operator. Call `getOwnerValidatorProxy`
+function by interacting with the NodeOperatorRegistry to retrieve the address of the ValidatorProxy that was created. 
+You will need to approve the NFT token to the retrieved address. After you've approved the token, call the `joinOperator` 
+function. That's it! You are now a part of the PoLido system!
+
+Guide:
+1. Approving the NFT token to ValidatorProxy
+
+   Visit https://remix.ethereum.org/, and create a new contract with an arbitrary name.
+
+   ![xd2imfoj79nwf4aa0eml](https://user-images.githubusercontent.com/17001801/153889832-90ed824e-f08f-4916-a77c-c59fc3245b3d.png)
+   
+   Paste the following code in the created contract:
+
+   ```solidity
+   // SPDX-FileCopyrightText: 2021 Shardlabs
+   // SPDX-License-Identifier: GPL-3.0
+   pragma solidity 0.8.7;
+
+   interface INodeOperatorRegistry {
+        function getOwnerValidatorProxy() external view returns (address);
+   }
+   ```
+   
+   Press CTRL + S to compile the code.
+
+   After that, click on Deploy & Run transactions and make sure that `INodeOperatorRegistry` is selected in the CONTRACT 
+   drop down list. Also, make sure that Injected Web3 is selected in the ENVIRONMENT drop down list and that ACCOUNT 
+   corresponds to the address that owns the Operator (you can change the connected account via MetaMask).
+
+   ![mgxkvjfkmjyys1mfsqi9](https://user-images.githubusercontent.com/17001801/153895814-77ba1f37-ee0d-4f35-828c-a9faf7e5dcf7.png)
+
+   Enter the address of the `NODE_OPERATOR_REGISTRY_PROXY` "0xb1f3f45360Cf0A30793e38C18dfefCD0d5136f9a" 
+   (the address in the image below is deprecated) in the field like from image below, and then click on the blue At 
+   `Address` button. This will provide you the interface to communicate with the NodeOperator.
+
+   ![fdunad10cz2y2fpnxp2o](https://user-images.githubusercontent.com/17001801/153897300-ffce2e6f-1fe3-47e9-b79e-80b2bfa60897.png)
+
+   Expand the `INODEOPERATORREGISTRY` interface from the bottom left corner of the Remix by clicking on the ">".
+
+   ![cvununqdej0wyk5oyqlc](https://user-images.githubusercontent.com/17001801/153898159-0fddef99-2468-448a-b70a-2afc7e3cbe20.png)
+
+   ![agr8kfprb94onqhp7eem](https://user-images.githubusercontent.com/17001801/153906028-e9f28cfd-a940-4eb3-b617-8a8b9a88214c.png)
+
```

Comment:
add the approve step to approve the NFT to the validatorProxy contract.
1- add ERC721 interface and approve function
2- use remix to show the users how to approve